File: CourseProjects/pythonRecap/classes.py
class Auto:
    def __init__(self, name, brand, year):
        self.name = name
        self.brand = brand
        self.year = year

    def startEngine(self):
        print('Engine started')


myCar = Auto('Mustang', 'Ford', 2010)

myCar.startEngine()

File: CourseProjects/pythonRecap/classes.js
class Car {
  constructor(name, brand, year) {
    this.name = name;
    this.brand = brand;
    this.year = year;
  }

  startEngine() {
    console.log('Engine started');
  }
}

const myCar = new Car('Mustang', 'Ford', 2010);
console.log(myCar);
